### PR TITLE
Skip `smoke_TestsROIPooling_*` f16 tests on ARM

### PR DESCRIPTION
**Pull request by vurusovs**

### CODE REVIEW

**Review by praasz — APPROVED** [review]

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp [hunk 1]
```diff
@@ -223,13 +223,14 @@ std::vector<std::string> disabledTestPatterns() {
 
 #if defined(OV_CPU_ARM_ENABLE_FP16)
     // Issue: 123019
-    retVector.emplace_back(R"(smoke_AvgPool_ExplicitPad_CeilRounding.*modelType=f16.*)");
```
Comment (ilyachur) [inline src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp:226]: Why did you remove this line?

Comment (vurusovs) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp:226]: Mistake, will return, thanks!

Comment (vurusovs) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp]: Fixed

**Comment by vurusovs** [comment]
`ie_tests_cpu_macos_arm64_release` passed

**Review by ilyachur — APPROVED** [review]
